### PR TITLE
menu: centralize menu and frontpage data

### PR DESCRIPTION
**Pull request by AoNoOokami**
* Adds a service to centralize data needed in menu and frontpage board.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>

## Why are you opening this PR?

To improve code maintenance.
https://tree.taiga.io/project/rero21-reroils/task/1316?kanban-status=1224895

## How to test?

Go to admin frontpage and check board behaviour. Check menu too.

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

File: projects/admin/src/app/frontpage/frontpage.component.html [hunk 1]
```diff
@@ -18,11 +18,10 @@
...
-  </div >
-    <admin-frontpage-board *ngIf="userServices && catalog && acquisitions && adminMonitoring"
-  [userServices]="userServices"
-  [catalog]="catalog"
-  [acquisitions]="acquisitions"
-  [adminMonitoring]="adminMonitoring">
-</admin-frontpage-board>
+  </div>
+  <div class="row">
+    <div *ngFor="let entry of linksMenu.entries; let i = index" [attr.data-index]="i" class="col-lg-3">
```
Comment (sebastiendeleze) [inline projects/admin/src/app/frontpage/frontpage.component.html]: What is the purpose of storing index in attr.data-index?

File: projects/admin/src/app/frontpage/frontpage.component.ts [hunk 2]
```diff
@@ -27,27 +27,26 @@ import { LocalStorageService } from '@rero/ng-core';
...
   ) { }
 
     /** On init hook */
   ngOnInit() {
-    this.initBoard();
+    this.initLinksMenu();
     this._localStorageService.onSet$.subscribe(() => {
-      const link = this.adminMonitoring.entries.find(
+      const link = this.linksMenu.entries.find(
+        (element: any) => element.name === 'Admin & Monitoring'
```
Comment (sebastiendeleze) [inline projects/admin/src/app/frontpage/frontpage.component.ts]: Is this value always the same or could be translated?

File: projects/admin/src/app/frontpage/frontpage.component.html (hunk 1 above)
Comment (AoNoOokami) [reply projects/admin/src/app/frontpage/frontpage.component.html]: Unused code, indeed.

File: projects/admin/src/app/frontpage/frontpage.component.ts (hunk 2 above)
Comment (AoNoOokami) [reply projects/admin/src/app/frontpage/frontpage.component.ts]: Indeed

**Pull request merged**